components/src/dynamo/sglang/request_handlers/handler_base.py

```
@@ -853,6 +861,111 @@ async def update_weight_version(self, body: dict) -> dict:
+                "status": "error",
+                "message": "elastic EP is not enabled (set --elastic-ep-backend)",
+            }
+
+        from sglang.srt.managers.io_struct import ScaleElasticEPReqInput
+
+        async with self._scale_ep_lock:
+            try:
+                result = await tokenizer_manager.scale_elastic_ep(
+                    ScaleElasticEPReqInput(new_ep_size=new_ep_size)
+                )
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)